README.md

```diff
@@ -50,6 +50,7 @@ If you have any questions, please join the #promscale channel on
   * [Data Retention](docs/sql_schema.md#data-retention)
 * **[Quick Tips](#-quick-tips)**
 * **[High Availability](docs/high-avaliability/prometheus-HA.md)**
+* **[Writing data to Promscale](docs/writing_to_promscale.md)**
```

Comment:
I would put this right after "Analyzing Data Using SQL". Also "data" should be "Data" (Capitalization)